adr/2020-11-06-konstraint-policy-library.md

```diff
@@ -0,0 +1,22 @@
+Alin:
+
+### Integrated Konstraint library to support Common Rego Language definitions.
```

Comment:
Please don't make this into a README. ADRs are supposed to show background/context against which decisions were made, so others don't have to revisit this decision process.

Please read the material on ADRs: https://adr.github.io/madr/#example

So please write down at least what the problem was, our explored options, and why we chose konstraint.